### ISSUE TITLE
Continuity between fuzzy dark matter and ALPs

### ISSUE THREAD

**Issue by cray0n** [lsstdarkmatter/dark-matter-graph#39]
I wonder if there is a way to make clear that these are in fact essentially the same but in different mass regimes and therefore with the quantum pressure playing a different role? Here is one related reference: https://arxiv.org/abs/astro-ph/0003365

(Chanda Prescod-Weinstein)

**Comment by yymao** [comment]
@cray0n thanks! This is a good point. In addition to adding the reference and reordering the models so that fuzzy and axion-like DM appear next to each other, maybe we want to add a "see also" section in the yaml file? And then we can render them in the sidebar and people can click on them.

**Comment by cray0n** [comment]
I like these ideas @yymao! Maybe I haven't explored the full capacity of the charts, but is it possible to highlight two things at once so that people can highlight all of the fuzzy and ALP boxes at the same time? If not, could this be added?

**Comment by yymao** [comment]
@cray0n Currently you can click an item, and then hover your mouse to a different item; in this case both (clicked path and hovered path) will be highlighted. 

In an older version of the web app, I actually implemented something similar to what you described --- when a user click two items, both paths will be highlighted (i.e., the second click does not clear existing highlights). But admittedly this behavior was a bit confusing. Maybe what we needed is to distinguish this behavior by using `ctrl`+click (i.e., when `ctrl` is pressed, click does not clear existing highlights). 

@kadrlica @cray0n, would this (the ability to highlight multiple paths by `ctrl`+click) be something useful? If so I can try to implement it.

**Comment by kadrlica** [comment]
`Ctrl+Click` could be super cool as a technical addition. 

I'm wondering though whether there is enough motivation to combing Fuzzy and Axion dark matter into a single box called "Ultra-light" dark matter. This would also have the effect of highlighting all boxes, and we could specify within the probe/measurement which types of models would be constrained. Input from @cray0n would be very welcome here!

**Comment by cray0n** [comment]
@kadrlica yeah I went back and forth about whether to propose that. (Putting on my science, technology, and society studies hat) Part of the issue is that people have typically focused on one mass regime OR the other but not both, partly because of diverse motivations for being interested in scalar dark matter candidates. Some people have a preference for halo-scale condensates (ultralight, 10^-22 eV), and other people (me) are motivated by Standard Model considerations (~ 10^-5 eV). They really do have different phenomenologies when you look at 20 or more orders of magnitude of difference. In my opinion, there actually is not enough generic work that looks at the middle ground where the quantum pressure has a different effect. 

So I think for now there is an argument for keeping them separate, unless we can find a way to bifurcate them in the chart, but that might be adding too much structure to the chart? We really need a 3D visualization :-P 

@yymao I think the added functionality would be useful, especially if we are going to keep them separate (as well as other things that might have crossover)

**Comment by yymao** [comment]
I split the technical part and created a new issue #41

**Comment by cray0n** [comment]
For clarity, @kadrlica @yymao I am open to push back on my view about the bifurcation!

tagging in @ahgpeter since she just wrote a nice long review with Matt Buckley where maybe they thought about this bifurcation?

**Comment by kadrlica** [comment]
"We really need a 3D visualization :-P" -- Be careful what you ask for @cray0n, @yymao might just do it!

**Comment by ahgpeter** [comment]
A few years ago, Richard Massey & others tried a 3D plot.  We never really got it to look right, despite much tinkering.  While I don’t think it’s impossible to do a beyond-2D plot, it is really hard to get right.  I like both the matrix and flow chart much better than any 3D plot I’ve seen.  Buckley & my 2D fig. 1 in our paper was our attempt to compress DM information to 2D for models.

A

**Comment by cray0n** [comment]
I was mostly joking about the 3D thing haha! But I haven't had a chance to really look through the paper yet and will inspect the figure you mention @ahgpeter, thanks.
